std_msgs/Header header
float64 ship_roll            # ship_roll reading
float64 ship_roll_variance   # ship_roll_variance reading
